share/spice/code_search/subtitle.handlebars

```diff
@@ -0,0 +1 @@
+<a href="repo_url" target="_blank">{{repo_url}}</a> - {{file_location}}
```

Comment:
Maybe link back to the searchcode page, this includes the script and link to repository. https://searchcode.com/codesearch/view/99010157